packages/config/src/Config.ts

```diff
@@ -82,6 +85,10 @@ function getSupportedPlatforms(
  * @param options enforce criteria for a project config
  */
 export function getConfig(projectRoot: string, options: GetConfigOptions = {}): ProjectConfig {
+  if (typeof options.useDynamicEval === 'undefined' && isDynamicEvalEnabled) {
```

Comment:
i think we should rename this to hot eval because we already call `config.js/ts` dynamic. so it's hot eval of dynamic config